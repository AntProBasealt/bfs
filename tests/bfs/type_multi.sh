bfs_diff links -type f,d,c
