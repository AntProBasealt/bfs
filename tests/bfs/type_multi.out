links
links/deeply
links/deeply/nested
links/deeply/nested/dir
links/deeply/nested/file
links/file
links/hardlink
